todo_app/data/trello_items.py

```diff
@@ -0,0 +1,66 @@
+from flask import Flask, render_template, request
+import os
+import requests
+import json
+from .TrelloClass import Item
+
+
+
+def get_trello_items():
+    url = "https://api.trello.com/1/boards/620e338747c98f4449c740c7/lists"
```

Comment:
This works but it would be nice to make the board id into an environment variable.

It's not sensitive but it is something we want to vary between environments - e.g. developers running the app against their own boards, a shared test environment and a "real" production environment.
It would also help with automated end-to-end testing.

And then the same applies to the two list IDs used lower down.

Reply:
Have made this a variable as per Jacks suggestion